roda-core/roda-core/src/main/java/org/roda/core/RodaCoreFactory.java

```diff
@@ -290,7 +290,7 @@ public Map<String, List<String>> load(Locale locale) {
           }
 
           List<String> properties = RodaCoreFactory
-            .getRodaConfigurationAsList("ui.sharedProperties.whitelist.messages.property");
```

Comment:
Not sure why this line changed. This will have implications! Please review this!